Rego/EXOConfig.rego

```diff
@@ -242,16 +242,25 @@ tests[{
     "ReportDetails" : ReportDetailsString(Status, ErrorMessage),
     "RequirementMet" : Status
 }] {
-    ErrorMessage := "Wildcard domain (\"*\") in shared domains list, enabling sharing with all domains by default"
-
-    Status := count(SharingPolicyAllowedSharing) == 0
+    ContactsSharingPolicies := SharingPolicyContactsAllowedAllDomains
+    ErrorMessage := Description(Format(ContactsSharingPolicies), "sharing polic(ies) are sharing contacts folders with all domains by default:", concat(", ", ContactsSharingPolicies))
+    Status := count(ContactsSharingPolicies) == 0
 }
 #--
 
 #
 # MS.EXO.6.2v1
 #--
 
+SharingPolicyCalendarAllowedAllDomains[SharingPolicy.Name] {
+    SharingPolicy := input.sharing_policy[_]
+    Domains = SharingPolicy.Domains[_]
+    WildcardInList := contains(Domains, "*")
+    CalendarInList := contains(Domains, "Calendar")
+    WildcardInList == true
+    CalendarInList == true
```

Comment:
Are we doing this compare this way for a reason? 
```
    contains(Domains, "*") == true
    contains(Domains, "Calendar") == true
```

Above would be more concise